content/chat/rooms/messages.textile

```diff
@@ -1,13 +1,14 @@
 ---
 title: Messages
-meta_description: "Send and receive messages in chat rooms."
+meta_description: "Send, update, delete and receive messages in chat rooms."
 product: chat
 languages:
   - javascript
   - react
 ---
 
-You can send and receive messages in a chat room with any number of participants. These users subscribe to messages by registering a listener, and send messages to all users that are subscribed to receive them.
+You can send, update, delete and receive messages in a chat room with any number of participants. These users subscribe to messages by registering a listener, and send messages to all users that are subscribed to receive them.
+A user can also update or delete a message, all users that are subscribed to the room will then receive the updated or deleted messages.
```

Comment:
What do we think about this wording update:

```Send, update, delete, and receive messages in a chat room with any number of participants. Users subscribe to messages by registering a listener, and send messages to all users that are subscribed to receive them.

A user can also update or delete a message, all users subscribed to the room will receive notifications of these changes.```

The last part didn't quite read right to me, but that may have just been me.

Reply:
Thanks @GregHolmes , I've reworded this a bit, please let me know what you think :)